editor/debugger/debug_adapter/debug_adapter_protocol.cpp

```diff
@@ -851,6 +851,11 @@ bool DebugAdapterProtocol::process_message(const String &p_text) {
 	Dictionary params = json.get_data();
 	bool completed = true;
 
+	// workaround for non-compliant DAP/JSON implementations, see #108288
```

Comment:
I'm not sure we need to explicitly say it's a workaround. I'd make the comment something more like:

```
// While JSON does not distinguish floats and ints, "seq" is an integer by specification. See #108288
```